cypress/integration/login.js

```diff
@@ -18,8 +20,48 @@ describe('Login functionality', function() {
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User')
+        cy.get('.user-info__title').contains(`${testUser}`)
         cy.get('.main-nav').find('li').should('have.length', 2)
         cy.get('.calendar--selected').contains(`${new Date().getDate()}`)
     })
+
+    describe('Should be able to login with all user roles and get required amount navigation tabs', () => {
```

Comment:
Is this nested describe really necessary? It works, but I wonder if a describe inside a describe looks nice. In any case, this does not hinder functionality.

Reply:
It is not necessary at all, however it contains iteration of similar tests, so in long run this might improve readability, when running tests. Just not sure if overall nesting describes is a good practise.